src/main/java/org/scijava/console/ConsoleService.java

```diff
@@ -62,6 +63,12 @@
 	/** Notifies listeners of output sent to {@code stdout} or {@code stderr}. */
 	void notifyListeners(OutputEvent event);
 
+	/** Stream that writes to stdout without notifying listeners. */
+	PrintStream getSystemOutBypassListeners();
```

Comment:
Ah, I see. You want the `StderrLogService` to write to stdout/stderr, but _without_ publishing `OutputEvent`s. I have mixed feelings about that.

The thing is: in general, listeners of `OutputEvent` want to know about everything that goes to the console—even stuff that originated from the `StderrLogService`. Just because _your_ use case (the fancy Log window) already knows about those messages from the logging side does not mean we should suppress the `OutputEvent` publication globally.

I assume your goal with the fancy log window is to avoid double printing the same message on both the default log channel and then consequently to the console, right? To achieve that, perhaps we should find a way to detect when an `OutputEvent` originated from a logging action, no? Or, the fancy log window could simply tell the truth about all this: if you have a separate tab for the console, it will also show the messages sent to the default logger, and that is OK! No harm done!